setup.py

```diff
@@ -27,13 +27,13 @@ def get_requirements(filename='requirements.txt'):
 
 setup(
     name='fmn.sse',
-    version='0.1.1',
+    version='0.2.0',
     description='Real time feed for fedmsg',
     long_description=get_description(),
-    author='Szymon M',
-    author_email='skrzepto@gmail.com',
+    author='Fedora Infrastructure Team and Szymon M',
```

Comment:
Maybe something like ``Szymon M, Jeremy Cline and the Fedora Infrastructure Team`` ? Or we could use the maintainer field.

Just food for thoughts here, I don't think it really matters